aten/src/ATen/native/sparse/SparseTensor.cpp

```diff
@@ -125,6 +125,13 @@ Tensor sparse_coo_tensor(ArrayRef<int64_t> size, const TensorOptions& options) {
   return at::_sparse_coo_tensor_with_dims(size.size(), 0, size, options.layout(at::kSparse));
 }
 
+Tensor& abs_out_sparse(Tensor& result, const Tensor& self) {
+  Tensor temp = self.coalesce();
+  Tensor temp_result = at::abs(temp.values());
+  get_sparse_impl(result)->resize_(temp.sparse_dim(), temp.dense_dim(), temp.sizes());
+  copy_into_sparse(result, temp.indices(), temp_result, false);
```

Comment:
I think you want non_blocking = `true` here, cuda operation should be non-synchronizing.

Reply:
will update the test casse

Reply:
will be putting the check for the same